src/test/java/com/hotels/bdp/circustrain/bigquery/util/SchemaUtils.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.circustrain.bigquery.util;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.when;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+
+import org.junit.Rule;
+
+import fm.last.commons.test.file.DataFolder;
+import fm.last.commons.test.file.RootDataFolder;
+
+import com.google.api.gax.paging.Page;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.Storage.BlobListOption;
+import com.google.common.io.Files;
+
+public class SchemaUtils {
```

Comment:
This is not a Util class test, it's just a helper class for your two actionTests, move it in the same package `com.hotels.bdp.circustrain.bigquery.extraction.container` as those tests and make everything in this class package private

Reply:
But `SchemaUtils` is used in `SchemaExtractorTest` as well. If I move it and make it package private, `SchemaExtractorTest` wouldn't be able to use it.

Unless I move SchemaExtractor to `com.hotels.bdp.circustrain.bigquery.extraction.container`, since it's used only by those schema action classes. Would that be good?

Reply:
right that reference didn't pop up in my IDE. It's ok let's leave it then